Readme.md

```diff
@@ -20,9 +20,9 @@ kubectl create ns kvrocks
 
 3. Use helm to install and manage the crd/operator 
 ```
-git clone https://github.com/RocksLabs/kvrocks-operator.git
-helm install kvrocks-crd deploy/crd -n kvrocks
-helm install kvrocks-operator deploy/operator -n kvrocks
+helm repo add kvrocks-operator https://rockslabs.github.io/kvrocks-operator
+helm install kvrocks-crd kvrocks-operator/kvrocks-crd -n kvrocks
```

Comment:
Hi, @ducviet00 

I think we can relocate the crd directory to the operator, similar to this example: https://github.com/jiayouxujin/helm-charts/tree/master/charts/kvrocks-operator.

Reply:
I think we should, but it is up to you. However, this change should not be included in this PR. 
btw, we should add an option to enable CRDs or not because I have faced some issues when working with GitOps tools like ArgoCD.
https://ray-project.github.io/kuberay/deploy/helm/#working-with-argo-cd